### PR TITLE
[SYCL] temporarily disable sycl-ls-unified-runtime.cpp test

### PR DESCRIPTION
**Pull request by cperkinsintel**
Universal Runtime will soon be changing its reporting.  Until it does so, this test is being disabled.

### CODE REVIEW

**Review by aelovikov-intel — APPROVED** [review]

File: SYCL/Plugin/sycl-ls-unified-runtime.cpp [hunk 1]
```diff
@@ -1,4 +1,5 @@
-// REQUIRES: level_zero
+// REQUIRES: TEMPORARY_DISABLED
+//  Universal Runtime will soon be changing its reporting.
```
Comment (smaslov-intel) [inline SYCL/Plugin/sycl-ls-unified-runtime.cpp]:
```suggestion
//  Unified Runtime will soon be changing its reporting.
```
:)

Comment (cperkinsintel) [reply SYCL/Plugin/sycl-ls-unified-runtime.cpp]: done

**Review by smaslov-intel — APPROVED** [review]

**Comment by againull** [comment]
Failures:
DeviceArchitecture/device_architecture.cpp
Basic/event_profiling_info.cpp

are unrelated and tracked. PR to disable these tests: https://github.com/intel/llvm-test-suite/pull/1504

**Pull request merged**